core/server/services/url/utils.js

```diff
@@ -303,7 +313,7 @@ function urlFor(context, data, absolute) {
         }
     } else if (context === 'api') {
         urlPath = getAdminUrl() || getBlogUrl();
-
+        let apiPath = getApiPath('stable');
```

Comment:
Again assumes `stable` to be default API if no data is passed to `urlFor` with context.

Reply:
I think that is fine, even though i like that you currently explicitly pass `stable:true` 👍